.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Use this template for reporting bugs encountered while using MLflow.
+labels: 'bug'
+title: [BUG]
+---
+Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md)
+for information on what types of issues we address.
+  
+Please do not delete this template unless you are sure your issue is outside its scope.
+
+### System information
+- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
+- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
+- **MLflow installed from (source or binary)**: 
+- **MLflow version (run ``mlflow --version``)**:
+- **Python version**: 
+- **npm version** (if running the dev UI):
```

Comment:
```suggestion
- **npm version** (if running the dev UI - leave blank otherwise):
```

Reply:
I think this is implied - feels a bit too wordy.